### ISSUE TITLE
Massive Issues with RTX 3080 Ti (0.25.8)

### ISSUE THREAD

**Issue by s0n1cx** [trexminer/T-Rex#1096]
Hi,

i'm getting only ~ 300MH/s @ALPH while dual mining with the RTX 3080 Ti. The temps are also higher. 

I got no issues with the previous Version 0.25.2 hitting ~ 600 MH/s

My Driver: 496.49

**Comment by sjcdcheu** [comment]
Overall, eth jumped from 450MH/s to 460MH/s LHR jumped from below 73 to 74.9% but my FHR drop a 2~3%.  ALPH jumped from 2G to 3GH/s.  My current EVGA 3080Ti ETH jumped from 85 to above 92 and ALPH current at 900MH/s.  Very good result.  My 3080Ti setting --lock-cclock 1850 --mclock 1600 --pl 79.  I am still using 472.12 driver and mining at 2miners.

**Comment by sunyao920807** [comment]
> 



> Overall, eth jumped from 450MH/s to 460MH/s LHR jumped from below 73 to 74.9% but my FHR drop a 2~3%. ALPH jumped from 2G to 3GH/s. My current EVGA 3080Ti ETH jumped from 85 to above 92 and ALPH current at 900MH/s. Very good result. My 3080Ti setting --lock-cclock 1850 --mclock 1600 --pl 79. I am still using 472.12 driver and mining at 2miners.

What is your current power consumption in watts?

**Comment by DanTheManOg** [comment]
Yeah same issues, my alph hashrate halfed with 25.8 on duel mining with a 3070ti. Just going to use 25.6 till this gets fixed.

**Comment by marcostrama** [comment]
For this new version same issues here when dual mining. It’s apparently ok for my 2 3080ti EVGA FTW3, I saw a small drop on ETH (80 to 76) but a great increase on ALPH (650 to 850). But for my 3080ti EVGA XC3 I saw drop on ETH and in ALPH (650 to less than 300)

**Comment by marcostrama** [comment]
> Yeah same issues, my alph hashrate halfed with 25.8 on duel mining with a 3070ti. Just going to use 25.6 till this gets fixed.

For me 25.6 is not appearing the ALPH hashs on HiveOS. I falled back to 25.2

**Comment by jeromeinfotech** [comment]
Only complaint - temperature is getting high (88-93c) - had to feed more power to get a better mining rates from 25.2 to 8 - actually getting good mining rates with ETH 70mhs and ALPH 1.05ghs on RTX3080 - updated driver too 511.79
